static/js/components/CommentEntry.jsx

```diff
@@ -62,6 +81,28 @@ const CommentEntry = ({ addComment }) => {
     addComment(data);
     reset();
     setGroupSelectVisible(false);
+    setTextValue("");
+    setAutosuggestVisible(false);
+    setUsernamePrefixMatches({});
+  };
+
+  const handleTextInputChange = (event) => {
+    const text = event.target.value;
+    const cursorIdx = event.target.selectionStart;
+    const currentWord = text.slice(0, cursorIdx).trim().split(" ").pop();
```

Comment:
Not 100% fool-proof (you can move the cursor, press Del to remove some chars, e.g.), but I think good enough for general editing.

Reply:
Right, I thought about that, but think it's fine if we only match from the "@" at the beginning of the word to where the cursor currently is -- users will quickly see that's what's being used for the matching in that case.